elixir.go

```diff
@@ -102,3 +105,50 @@ func authenticateWithOidc(oauth2Config oauth2.Config, provider *oidc.Provider, c
 	return idStruct, err
 
 }
+
+// Returns long-lived token as string
+func generateJwtFromElixir(tokenElixir, key, alg string) string {
+	var (
+		elixirClaims   jwt.MapClaims
+		EGAtokenString string
+	)
+
+	token, _ := jwt.Parse(tokenElixir, func(tokenElixir *jwt.Token) (interface{}, error) { return nil, nil })
+	if claims, ok := token.Claims.(jwt.MapClaims); ok {
+		elixirClaims = claims
+	} else {
+		log.Error("Claims in token are empty")
+	}
+
+	ttl := 170 * time.Hour
+	elixirClaims["exp"] = time.Now().UTC().Add(ttl).Unix()
+	EGAtoken := jwt.NewWithClaims(jwt.GetSigningMethod(alg), token.Claims)
+	EGAtoken.Header = token.Header
+	data, err := ioutil.ReadFile(key)
+	if err != nil {
+		log.Fatal(err, data)
```

Comment:
```suggestion
		return nil, err
```
It is better to return the error to main and do the handling from there.

Reply:
Good comments, I should pay more attention on these kind of stuff.